# 单个检测到的物体信息
geometry_msgs/Pose pose
string label
float32 confidence
int32 class_id